Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -264,7 +264,7 @@
                     builder
                         .withDataSourceName("ActiveAccordingToLifeCycle")
                         .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Boolean)
-                        .withTitle("Status ifølge Status")
+                        .withTitle("Status (Livscyklus)")
```

Comment:
default with for this one needs to be fixed so fit on one line: 
![image](https://user-images.githubusercontent.com/13116889/191694342-ee2b5756-590c-405a-b0a6-0ddbaa03dcd0.png)